stdlib/Random/docs/src/index.md

```diff
@@ -8,9 +8,12 @@ Random number generation in Julia uses the [Xoshiro256++](https://prng.di.unimi.
 by default, with per-`Task` state.
 Other RNG types can be plugged in by inheriting the `AbstractRNG` type; they can then be used to
 obtain multiple streams of random numbers.
-Besides the default `TaskLocalRNG` type, the `Random` package also provides `MersenneTwister`,
-`RandomDevice` (which exposes OS-provided entropy), and `Xoshiro` (for explicitly-managed
-Xoshiro256++ streams).
+
+The types exported by the `Random` package are:
+* `TaskLocalRNG`: the default with 1 stream per task.
+* `Xoshiro`: for explicitly-managed Xoshiro256++ streams.
+* `RandomDevice`: for OS-provided entropy. This should be used for cryptographically secure random numbers (CPR.
+* `MersenneTwister`: for backward compatibility (usually slower and worse quality than Xoshiro).
```

Comment:
```suggestion
* `MersenneTwister`: an alternate high-quality PRNG which was the default in older versions of Julia, and is also quite fast, but requires much more space to store the state vector and generate a random sequence.
```

Reply:
> but requires much more space to store the state vector and generate a random sequence.

Maybe simply "but requires much more space to store the state" ? Or "... state vector" (although technically there are multiple vectors).